.gitignore

```diff
@@ -18,3 +18,6 @@
 
 # Ignore vim backup files
 *.swp
+
+# Ignore schema.rb file
+db/schema.rb
```

Comment:
Why would you ignore the schema file! We need that to synchronise our database. It would break `rake db:setup` and make it harder to deploy at the end of the week.